src/net.cpp

```diff
@@ -2471,54 +2460,78 @@ void CConnman::ThreadOpenMasternodeConnections()
                             if (nANow - lastAttempt < chainParams.LLMQConnectionRetryTimeout()) {
                                 continue;
                             }
-                            pending.emplace_back(dmn);
+                            ret.emplace_back(dmn);
                         }
                     }
                 }
+            return ret;
+        };
+
+        const auto getPendingProbes = [&]() {
+            LockAssertion lock(cs_vPendingMasternodes);
+            std::vector<CDeterministicMNCPtr> pending;
```

Comment:
```suggestion
            std::vector<CDeterministicMNCPtr> ret;
```
same as in `getPendingQuorumNodes()`